test/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:18.04
+RUN apt-get update && apt-get -yy install curl openssl
+
+# Install NVM
+SHELL ["/bin/bash", "-c"]
+# nvm environment variables
+ENV NVM_DIR /usr/local/nvm
+ENV NODE_VERSION 16.14.0
+
+# install nvm
+# https://github.com/creationix/nvm#install-script
+RUN curl --silent -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.2/install.sh | bash
+
+# install node and npm
+RUN source $NVM_DIR/nvm.sh \
+    && nvm install $NODE_VERSION \
+    && nvm alias default $NODE_VERSION \
+    && nvm use default
+
+# add node and npm to path so the commands are available
+ENV NODE_PATH $NVM_DIR/v$NODE_VERSION/lib/node_modules
+ENV PATH $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
+
+# confirm installation
+RUN node -v
+RUN npm -v
+SHELL ["/bin/sh", "-c"]
+
+# Download and install the full service package
+ENV mirversion="1.5.0"
```

Comment:
This was intentional. I felt that it was better for our testing to run against the binary we actually provide to our clients rather than building it separately since that allows to to also ensure that we built the binary correctly when releasing. I think we could provide building from the script as an option also if you like.